tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj

```diff
@@ -671,7 +671,7 @@
 				"${BUILT_PRODUCTS_DIR}/PromisesObjC/FBLPromises.framework",
 				"${BUILT_PRODUCTS_DIR}/mopub-ios-sdk/MoPubSDK.framework",
 				"${BUILT_PRODUCTS_DIR}/nanopb/nanopb.framework",
-				"${PODS_XCFRAMEWORKS_BUILD_DIR}/OMSDK_Mopub/OMSDK_Mopub.framework/OMSDK_Mopub",
```

Comment:
should it be here?

Reply:
after I reverted it and used "pod install" again this was modified again by pod install.